CHANGELOG.md

```diff
@@ -4,13 +4,32 @@
 
 - Feature:
     - Support collecting and reporting logs to backend (#147)
+    - Support profiling Python method level performance (#127
     - Add a new `sw-python` CLI that enables agent non-intrusive integration (#156)
-
-- New plugins:
-    - Falcon Plugin (#146)
+    - Add exponential reconnection backoff strategy when OAP is down (#157)
+    - Suport ignoring traces by http method (#143)
+    - `NoopSpan` on queue full, propagation downstream (#141)
+    - Support agent namespace. (#126)
+    - Added `grpc.max_connection_age_grace_ms` (#130)
```

Comment:
This is no longer present, it was an attempt to solve what turned out to be the nginx grpc proxy problem but didn't work. Root cause was fixed later.